main.go

```diff
@@ -128,6 +130,7 @@ func main() {
 		"leader-election-retry-period is the duration the LeaderElector clients should wait between tries of actions. Default is 2 seconds.")
 	flag.DurationVar(&controllerCacheSyncTimeout, "controller-cache-sync-timeout", defaultControllerCacheSyncTimeout, "CacheSyncTimeout refers to the time limit set to wait for syncing caches. Defaults to 2 minutes if not set.")
 	flag.DurationVar(&webhookInitializeTimeout, "webhook-initialize-timeout", defaultWebhookInitializeTimeout, "WebhookInitializeTimeout refers to the time limit set to wait for webhook initialization. Defaults to 60 seconds if not set.")
+	flag.IntVar(&defaultTtlForAlwaysNodeimage, "default-ttl-for-always-nodeimage", defaultTtlForAlwaysNodeimageConst, "DefaultTtlForAlwaysNodeimage refers to the calculation of the time limit the lifetime of a pulling task that has finished execution. Defaults to 300 seconds if not set.")
```

Comment:
if value is not time.Duration, then we should be more explicit that what unit default-ttl-for-always-nodeimage use. Consider rename it to default-ttlseconds-for-always-nodeimage